src/views/blocks/attendees/view-link.php

```diff
@@ -58,7 +59,8 @@
 
 $link = $view->get_tickets_page_url( $event_id, $is_event_page );
 
-$message = esc_html( sprintf( __( 'You have %1s for this %2s.', 'event-tickets' ), implode( _x( ' and ', 'separator if there are both RSVPs and Tickets', 'event-tickets' ), $counters ), $events_label_singular ) );
+// Translators: 1: number of RSVPs and/or Tickets with accompanying ticket type text, 2: post type label
+$message = esc_html( sprintf( __( 'You have %1s for this %2s.', 'event-tickets' ), implode( _x( ' and ', 'separator if there are both RSVPs and Tickets', 'event-tickets' ), $counters ), $post_type_singular ) );
```

Comment:
:no_entry_sign: Multiple placeholders should be ordered. Expected '%1$1s, %2$2s', but got %1s, %2s.

<sub>hash: d24ea5835aadae934c15db3ef7e9fb53</sub>